websites/docs-smoke-test/src/content/components/multi-language-code-examples.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Multi-Language Code Examples
+---
+
+# JavaScript, Java, Console
+
+<MultiLanguageCodeExamples title="Multilanguage Code Samples">
```

Comment:
Why is the name of the component not the one defined in the issue text?   The requirements described this:

```jsx
<MultiCode title="Creating a Cart">
  .... 
</MultiCode>
```
We wanted a short and clear enough name so the author's MDX stays easy on the eye.  The MDX-Facing  component names are optimized for being simple and easy to remember and not for being perfectly disjunct like you would make the names inside a bigger react codebase.

Reply:
Yeah makes sense.

Also to make it less error prone, we can also make it very explicit which child components should be rendered inside the `MultiCode`:

```
<MultiCode title="Creating a Cart">
  <MultiCode.Example path="create-cart.js" />
  <MultiCode.Example path="create-cart.java" />
</MultiCode>
```

The child components are static exports of the `MultiCode` component. This is a "common" thing to do in React when using compound components.

See:
- https://github.com/commercetools/ui-kit/tree/master/src/components/inputs/radio-input
- https://kentcdodds.com/blog/compound-components-with-react-hooks

Reply:
@emmenko That's intutive and logical for a React dev but I don't think it's useful for others.  I prefer to have them to write the exact same component name inside a multicode that they write for a single one (if it does not lead to a horrible implementation)

Reply:
We can still have the implementation like this and expose the component to MDX under a different name.

Reply:
If that works, fine.  I have a hard time though imagining how to expose two different <CodeExample>  components sunder the same name to MDX.